templates/profiles/membership.html

```diff
@@ -57,6 +57,12 @@ <h3>Medlemskap</h3>
                             studentweb.</p>
                         {{ field_of_study_application.documentation|as_crispy_field }}
                     </div>
+                    <div id="field-of-study-application-master-documentation">
+                        <p>For å søke master medlemsskap i linjeforeningen, kreves det at det gis et bevis på opptak.
```

Comment:
"Medlemskap" is the correct way to write this in Norwegian[0] :D

Also, "master medlemskap" is a nice "orddelingsfeil", so maybe even rewrite it to "medlemskap som masterstudent"?

You can probably also drop the comma in this sentence to make it flow a bit better.

0: http://www.sprakradet.no/svardatabase/sporsmal-og-svar/medlemskap-eller-medlemsskap/